### PR TITLE
Add note on requirements (closes #22)

### LINKED ISSUE #22: problems installing requirements
Stack trace:
```
/var/folders/bn/mw_hfq3n42lbt_vpd7h4rhfsfplj_1/T/H5close6uqqrouf.c:1:1: warning: type specifier missing, defaults to 'int' [-Wimplicit-int]
    main (int argc, char **argv) {
    ^
    /var/folders/bn/mw_hfq3n42lbt_vpd7h4rhfsfplj_1/T/H5close6uqqrouf.c:2:5: warning: implicit declaration of function 'H5close' is invalid in C99 [-Wimplicit-function-declaration]
        H5close();
        ^
    2 warnings generated.
    ld: library not found for -lhdf5
    clang: error: linker command failed with exit code 1 (use -v to see invocation)
    * Using Python 3.7.0 (v3.7.0:1bf9cc5093, Jun 26 2018, 23:26:24)
    * USE_PKGCONFIG: True
    .. ERROR:: Could not find a local HDF5 installation.
       You may need to explicitly state where your local HDF5 headers and
       library can be found by setting the ``HDF5_DIR`` environment
       variable or by using the ``--hdf5`` command-line option.
```

Do I need some kind of HDF5 installation outside of Python? (Apologies if this is a naive question).

### PR DESCRIPTION
**Pull request by natanlao**

### CODE REVIEW

File: Explore an HCA Data Set in Scanpy (Nov 2018)/README.md [hunk 1]
```diff
@@ -16,6 +16,11 @@ distribution. For some users, it may be necessary to use `pip3` instead.
 pip install -r ./REQUIREMENTS.txt
```
Comment (chmreid) [inline Explore an HCA Data Set in Scanpy (Nov 2018)/README.md]: Should change this to `pip install -r requirements.txt` while we're at it

File: Explore an HCA Data Set in Scanpy (Nov 2018)/README.md [hunk 2]
```diff
@@ -16,6 +16,11 @@ distribution. For some users, it may be necessary to use `pip3` instead.
...
 ```
 
+Depending on your platform, you might need to install additional depenencies:
+
+    $ sudo apt-get install libblas-dev liblapack-dev libatlas-base-dev gfortran libhdf5-dev
```
Comment (chmreid) [inline Explore an HCA Data Set in Scanpy (Nov 2018)/README.md]: What should users with non-Debian distros (e.g., Mac homebrew users or Fedora yum users) do?

File: Explore an HCA Data Set in Scanpy (Nov 2018)/README.md [hunk 3]
```diff
@@ -16,6 +16,11 @@ distribution. For some users, it may be necessary to use `pip3` instead.
...
+
+
 ## Usage
 
 Start up the jupyter kernel, which will open a web browser. You can then
```
Comment (chmreid) [inline Explore an HCA Data Set in Scanpy (Nov 2018)/README.md]: Should change this to "start up the jupyter kernel using the command below" to be more clear

File: Explore an HCA Data Set in Scanpy (Nov 2018)/README.md (hunk 2 above)
Comment (natanlao) [reply Explore an HCA Data Set in Scanpy (Nov 2018)/README.md]: Punting this to next sprint per discussion

**Pull request merged**